### ISSUE TITLE
lua style counted for

### ISSUE THREAD

**Issue by severak** [yosbelms/cor#14]
in [lua](https://en.wikipedia.org/wiki/Lua_%28programming_language%29), there is another `for` cycle flavour, eg.:

```
-- counting from 1 up to 10
for i=1,10 do
  print(i)
end
```

It's something similar possible in Cor?

**Comment by yosbelms** [comment]
Actually Cor does not support range iterators, just FOR and FOR-IN see the [docs](http://yosbelms.github.io/cor/docs/documentation.html#forloop). However the issue will remain open.

**Comment by yosbelms** [comment]
@severak I thought of this `for` flavor, my first proposal is:

```
for i in 1:10 {
    //something
}
```

what dou you think about?

PS: In Cor ranges has the syntax `n:m`, example: 

```
array = [1, 2, 3, 4]
slice = array[0:3]
```

**Comment by severak** [comment]
@yosbelms: that looks great

**Comment by yosbelms** [comment]
soon in the next release

**Comment by yosbelms** [comment]
it has been addressed in a8fae3e574e629a9ab62b87af1d19ce6c84ce092
